packages/change-codemod/transforms/add-require-for-global-identifier.js

```diff
@@ -2,13 +2,23 @@
 // top of the file.  Useful when needing to add requires for global identifiers.
 //
 // The identifier must be specified using the --identifier option, e.g. --identifier=sinon
+// The argument for the new require statement can be specified separately from the targeting identifier, e.g. --identifier=_ --packageName=lodash
 export default function transformer(file, api, options) {
   let { identifier } = options;
   if (!identifier) throw new Error("Please specify the name of the identifier name you would like to detect using the --identifier option.");
 
+  const packageName = options.packageName || identifier;
   const j = api.jscodeshift;
   const root = j(file.source);
+
+  const alreadyHasRequireDeclaration =
+    root
+      .find(j.CallExpression, { callee: { name: 'require' } })
+      .filter(n => n.node.arguments.length >= 1 && n.node.arguments[0].value === packageName)
+      .size() > 0;
+
   const found =
```

Comment:
You can just return here if `alreadyHasRequireDeclaration` since we won't need to do anything further.